File: tracks/python/rna_transcription.py
def to_rna(dna_strand):
    rna_letters = {'G': 'C', 'C': 'G', 'T': 'A', 'A': 'U'}
    rna_strand = ""
    for letter in dna_strand:
        if letter in rna_letters:
            rna_strand += rna_letters[letter]
    return rna_strand


print(to_rna("ACGTGGTCTTAA"))

File: tracks/javascript/rna-transcription.js
export const toRna = dnaStrand => {
    let rnaStrand = "";
    for (const dna of dnaStrand) {
        switch (dna) {
            case "G":
                rnaStrand += "C";
                break;
            case "C":
                rnaStrand += "G";
                break;
            case "T":
                rnaStrand += "A";
                break;
            case "A":
                rnaStrand += "U";
                break;
        }
    }
    return rnaStrand;
};

console.log(toRna("ACGTGGTCTTAA"));